### PR TITLE
Upload files

### PR DESCRIPTION
**Pull request by jpiechowka**

### CODE REVIEW

File: src/CujojsClient.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const ctxImpl = new ExplicitContext();
+const recorder = new BatchRecorder({
+    logger: new HttpLogger({
+        endpoint: 'http://localhost:9411/api/v1/spans'
+    })
+});
+
+const tracer = new Tracer({ctxImpl, recorder});
+const nameOfRemoteService = 'cujojs-client';
```
Comment (eirslett) [inline src/CujojsClient.js:15]: Are you sure this is correct?

**Comment by jpiechowka** [comment]
@eirslett 
This is what I used:
[Zipkin-js-example](https://github.com/openzipkin/zipkin-js-example/blob/initial/browser.js)
[Zipkin-instrumentation-cujojs-rest](https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-cujojs-rest/README.md)

It works for me, the only issue with this right now is this, so I used dirty workaround of adding timeout:
[Zipkin-js/issues/33](https://github.com/openzipkin/zipkin-js/issues/33)

**Comment by eirslett** [comment]
I was unsure about the part "const nameOfRemoteService = 'cujojs-client';". I would suppose cujojs-client is the name of the local service, not the remote one?

**Comment by jpiechowka** [comment]
@eirslett Oh, yes it is name of the local service - the client. There is a screenshot in the repo if you want to see how it looks in Zipkin.

**Comment by eirslett** [comment]
Yes, that's why I thought maybe a name like "const nameOfClient = 'cujojs-client';" would be more descriptive?

**Comment by codefromthecrypt** [comment]
+1

**Comment by jpiechowka** [comment]
@eirslett I have changed `nameOfRemoteService` to `nameOfClient`.

**Pull request merged**